### DISCUSSION TITLE
BeginAtZero and Steps in Radar Chart type

### DISCUSSION THREAD

**Discussion by sikelio** [erossini/BlazorChartjs#72] [Q&A] [answered]
Hi,
I wanted to know if the BeginAtZero and Step are implemented in Radar Chart type.
I have read the doc and try to find something in the source code and I didn't find something about it for the radar chart type.

Here is a screenshot of what I'm trying to implement

![image](https://github.com/erossini/BlazorChartjs/assets/40485515/29015d0e-1da9-4eff-b9ea-ec0299dcba97)

And here is the current result with PSC.Blazor.Components.Chartjs

![image](https://github.com/erossini/BlazorChartjs/assets/40485515/4c04cf04-0816-417b-b1e9-2892adedd6d8)

If someone could give me anwser ^^

**Comment by sikelio** [comment, marked as answer]
I found my response by trying to add it ine the package, I'll make an PR

**Comment by erossini** [comment]
Thank you for your comment. Soon, I'm realising the new version of the component in `NET8`.
Enrico

**Reply by sikelio** [reply]
No problem :)
I've seen the PR for .NET8 update in the waiting list, I can't wait for it ^^
